lib/msf/core/post/linux/compile.rb

```diff
@@ -11,35 +11,43 @@ def initialize(info = {})
     super
     register_options( [
       OptEnum.new('COMPILE', [true, 'Compile on target', 'Auto', ['Auto', 'True', 'False']]),
+      OptEnum.new('COMPILER', [true, 'Compiler to use on target', 'gcc', ['gcc', 'clang']]),
```

Comment:
It'd be really nice of there was also an `Auto` option here to just find a usable compiler.

Reply:
Absolutely, this will be in a follow-up PR :)